src/mmrelay/plugin_loader.py

```
@@ -884,311 +884,543 @@ def _raise_install_error(pkg_name):
+                return _update_existing_repo_to_commit(repo_path, ref_value, repo_name)
+            else:
+                return _update_existing_repo_to_branch_or_tag(
+                    repo_path,
+                    ref_type,
+                    ref_value,
+                    repo_name,
+                    is_default_branch,
+                    default_branches,
+                    repo_url,
+                )
+    except (subprocess.CalledProcessError, FileNotFoundError) as e:
```

Ruff (0.14.3)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)